src/eduid/webapp/common/api/decorators.py

```diff
@@ -116,6 +119,16 @@ def verify_identity_decorator(*args: Any, **kwargs: Any) -> EduidViewReturnType:
         if isinstance(locked_nin, NinIdentity) and locked_nin.number != kwargs["nin"]:
             logger.info("User has a different locked NIN")
             logger.debug(f"Locked NIN: {locked_nin.number}. New NIN: {kwargs['nin']}")
+            if isinstance(session.app, MobileProofingApp | LetterProofingApp | OIDCProofingApp):
```

Comment:
This might be unnecessary as it is only there three apps that use this decoration

Reply:
Yeah and it's only LetterProofingApp that is in used in prod today, all other proofing apps does not require the user to input nin. I guess the future of this decorator should be to be able to be used in all proofing apps or get removed. So I think we can keep it the way you did it for now.